holopy/scattering/interface.py

```diff
@@ -134,7 +134,7 @@ def determine_default_theory_for(scatterer):
             theory = Mie()
     elif isinstance(scatterer, Spheroid) or isinstance(scatterer, Cylinder):
         theory = Tmatrix()
-    elif DDA().can_handle(scatterer):
+    elif DDA.can_handle(scatterer):
```

Comment:
good catch!

Reply:
Thanks! It helps avoid the DDA instantiation, which can cause crashes because of our suppressing of fortran output. 

The other "catch" here was that the `DDA.can_handle` used to return True always, whereas now it returns `isinstance(scatterer, Scatterer)`.